modules/porous_flow/test/tests/pressure_pulse/tests

```diff
@@ -7,7 +7,7 @@
     threading = '!pthreads'
     issues = '#6845'
     design = 'porous_flow/tests/pressure_pulse/pressure_pulse_tests.md'
-    requirement = 'PorousFlow shall correctly simulate the transient evolution of a pressure pulse in 1D using 1 phase physics'
+    requirement = 'PorousFlow shall correctly simulate the transient evolution of a pressure pulse in 1D using 1 phase physics.'
```

Comment:
```suggestion
    requirement = 'The porous flow module shall correctly simulate the transient evolution of a pressure pulse in 1D using 1 phase physics.'
```
Similarly for remaining file mods

Reply:
As above, I did a bulk s/PorousFlow /The porous flow module / on this file. I had a quick look and this that it all reads ok.